alien-root-legacy.sh

```diff
@@ -55,6 +58,7 @@ module load BASE/1.0 ${GCC_TOOLCHAIN_VERSION:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSI
 
 # Our environment
 setenv ALIEN_ROOT_LEGACY_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+setenv GSHELL_ROOT \$::env(ALIEN_ROOT_LEGACY_ROOT)
```

Comment:
Please add GSHELL_NO_GCC=1 in the Modulefile as well, thanks!

Reply:
Done.